jscomp/ext/ext_js_regex.ml

```diff
@@ -0,0 +1,20 @@
+(*It is unfortunate we cannot use a recursive function to process
+string and for loop is very limited in OCaml *)
+let check_from_end s =
+    let rtn = ref false in 
+    let () = for i = String.length s - 1 downto 0 do
+        let c = String.get s i in 
+        if c = '/' && i <> 0 then
+            rtn := true
+    done in 
+    !rtn
+
```

Comment:
Some tips: you don't need walk through the whole string though, you can learn from `List.exists` how it is implemented